x-pack/plugins/ml/public/timeseriesexplorer/_timeseriesexplorer_annotations.scss

```diff
@@ -0,0 +1,69 @@
+// SASS TODO: This uses non-BEM styles to be in line with the existing
+// legacy Time Series Viewer style. Where applicable it tries to avoid
+// overrides. The one override with `.extent` is because of d3.
+.ml-annotation-brush .extent {
+  stroke: #999;
+  stroke-width: 2px;
+  stroke-dasharray: 2 5;
+  fill-opacity: .05;
+  shape-rendering: geometricPrecision;
+}
+
+.ml-annotation-rect {
+  stroke: #000;
+  stroke-width: 1.5;
+  transition: stroke-width 125ms;
+  stroke-opacity: 0.2;
+  fill: #000;
+  transition: fill 125ms;
+  fill-opacity: 0.05;
+  transition: fill-opacity 125ms;
+  shape-rendering: geometricPrecision;
+  border-radius: 5px 5px 0 0;
+}
+
+.ml-annotation-rect-highlight {
+  fill: #000;
+  transition: fill 125ms;
+  fill-opacity: 0.1;
+  transition: fill-opacity 125ms;
+  stroke-opacity: 0.3;
+  transition: stroke-opacity 125ms;
+}
+
+.ml-annotation-rect-blur {
+  fill-opacity: 0.03;
+  transition: fill-opacity 125ms;
+  stroke-opacity: 0.1;
+  transition: stroke-opacity 125ms;
+}
+
+.ml-annotation-text {
+  text-anchor: middle;
+  font-size: 12px;
+  font-family: "Open Sans", Helvetica, Arial, sans-serif;
+  font-weight: 600;  
+  fill: #000;
+  transition: fill 125ms;
+}
+
+.ml-annotation-text-blur {
+  fill: #999;
+  transition: fill 125ms;
+}
+
+.ml-annotation-text-rect {
```

Comment:
You should try and use eui color variables here

Reply:
Done in 903b999.